pkg/cluster/disableupdates.go

```diff
@@ -20,6 +22,18 @@ func (m *manager) disableUpdates(ctx context.Context) error {
 		cv.Spec.Upstream = ""
 		cv.Spec.Channel = ""
 
+		// when installing via Hive we replace the quay.io domain with the ACR domain, so now we set it back to the expected domain
+		if m.installViaHive {
+			version, err := m.openShiftVersionFromVersion(ctx)
+			if err != nil {
+				return err
+			}
+			cv.Spec.DesiredUpdate = &configv1.Update{
+				Version: version.Properties.Version,
+				Image:   strings.Replace(version.Properties.OpenShiftPullspec, m.env.ACRDomain(), "quay.io", 1),
+			}
```

Comment:
is there a version change when we change the image?

Reply:
Potentially, yes. I did not test that though.

Reply:
Would be nice if you could test it

Reply:
From the doc, it seems it is relevant only when doing upgrade. However we are doing install. So we indeed might just be able to skip this field.

Reply:
@facchettos I certainly didn't have time to do so last week, but I'll look after my shift today.

Reply:
As expected its functional, but not a great user experience, because the `Version` in `DesiredUpdate` is an empty string as a result.
```yaml
$ oc get clusterversion -oyaml
apiVersion: v1
items:
- apiVersion: config.openshift.io/v1
  kind: ClusterVersion
...snip...
  spec:
    clusterID: 59568119-41e1-4d57-a513-a988d527ac6a
    desiredUpdate:
      force: false
      image: quay.io/openshift-release-dev/ocp-release@sha256:58a68bea6f4d5c23a10596953f50529417b33c331195deb9041d9fecfa95b400
      version: ""
  status:
    availableUpdates: null
...snip...
```
This should be left as it is, so that the expected version is listed.

Reply:
indeed